bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/EncryptionMediator.cs

```diff
@@ -41,7 +41,7 @@ public class EncryptionMediator : DisposableBase
             PlaintextBallot plaintext, bool verifyProofs = false)
         {
             // TODO: default to usePrecomputedValues: false
```

Comment:
should we remove the todo here since you made the change? i think this was an artifact left over from november

Reply:
Removed